pages/blog/page/[page].js

```diff
@@ -0,0 +1,54 @@
+import { PageSeo } from '@/components/SEO'
+import siteMetadata from '@/data/siteMetadata'
+import { getAllFilesFrontMatter } from '@/lib/mdx'
+import BlogLayout from '@/layouts/BlogLayout'
+import { POSTS_PER_PAGE } from '../../blog'
+
+export async function getStaticPaths() {
+  const totalPosts = await getAllFilesFrontMatter('blog')
+  const totalPages = Math.ceil(totalPosts.length / POSTS_PER_PAGE)
+  const paths = Array.from({ length: totalPages - 1 }, (_, i) => ({
```

Comment:
`/blog/1` should return the same as `/blog`. Currently, it leads to a 404.

Reply:
I achieved this by using the Next.js [Redirect API](https://nextjs.org/docs/api-reference/next.config.js/redirects) and have set the redirect to be `301 Moved Permanently` since it's [better for SEO](https://moz.com/learn/seo/redirection).

Reply:
I am actually thinking of repeating the page with the canonical link set to `/blog`

Reply:
This should work now but, because the blog have 8 posts and I set `POSTS_PER_PAGE` to 10, `blog/1` return 404.
Once the blog have more than 10 posts, it will work fine.